src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
```

Comment:
1. The `throwError` method intentionally throws a `RuntimeException`. While this might be useful for testing and debugging, make sure such deliberate errors are handled properly in production code.